src/pipecat/processors/frameworks/rtvi_helpers/llm.py

```diff
@@ -0,0 +1,210 @@
+import sys
+from enum import Enum
+from typing import Any, Dict, List
+
+from openai._types import NotGiven
+from pydantic import BaseModel
+
+from pipecat.frames.frames import (
+    LLMMessagesAppendFrame,
+    LLMMessagesUpdateFrame,
+    LLMSetToolsFrame,
+)
+from pipecat.processors.aggregators.llm_response import (
+    LLMUserContextAggregator,
+)
+from pipecat.processors.frameworks.rtvi import (
+    ActionResult,
+    RTVIAction,
+    RTVIActionArgument,
+    RTVIProcessor,
+)
+
+if sys.version_info < (3, 11):
+
+    class StrEnum(str, Enum):
+        """A string-based Enum class for Python versions < 3.11."""
+        def __new__(cls, value):
+            """Constructor for StrEnum."""
+            obj = str.__new__(cls, value)
+            obj._value_ = value
+            return obj
+else:
+    from enum import StrEnum
+
+
+class RTVILLMActionType(StrEnum):
+    """Enum for RTVI LLM action types."""
+    APPEND_TO_MESSAGES = "append_to_messages",
+    GET_CONTEXT = "get_context",
+    SET_CONTEXT = "set_context",
+    RUN = "run",
+
+
+class RTVIHelper(BaseModel):
```

Comment:
if/when we introduce other types of helpers (tts, stt), I can move this to a separate file